### PR TITLE
Time Sensitive | Elasticsearch 7 and LES Updates | LRDOCS-5912

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-5912, this should be merged as soon as possible. Sorry for the last-minute push, this stuff was changing under our feet over the last week. cc @lipusz

### CODE REVIEW

**Comment by rbohl** [comment]
Ahh, wait a minute Rich. There's a bug we need to doc. :hourglass_flowing_sand:

**Comment by rbohl** [comment]
Okay, merge away, @sez11a !

File: en/deployment/articles-dxp/03-installing-a-search-engine/03-enterprise-search/01-les-intro.markdown [hunk 1]
```diff
@@ -6,46 +6,68 @@ header-id: installing-liferay-enterprise-search
...
+beyond what's available out of the box with your @product@ subscription. It
+includes
+
+- Connector to use X-Pack Security*
+- Connector to use X-Pack Monitoring
+- Learning to Rank
+
+* A LES subscription is not necessary if using Elasticsearch 7 via the _Liferay Connector to
+    Elasticsearch 7_. X-Pack's security features are bundled. See the [LES
+    compatibility matrix]()
```
Comment (sez11a) [inline en/deployment/articles-dxp/03-installing-a-search-engine/03-enterprise-search/01-les-intro.markdown]: You're missing a link to the compatibility matrix here.

File: en/deployment/articles-dxp/03-installing-a-search-engine/03-enterprise-search/02-monitoring/01-monitoring-intro.markdown [hunk 2]
```diff
@@ -8,12 +8,20 @@ header-id: installing-liferay-enterprise-search-monitoring
...
-with X-Pack), and @product@ can communicate effortlessly and securely. A Liferay
-Enterprise Search Standard subscription (included with Premium) is necessary for
-this integration.  Contact 
+server) with X-Pack so that Elasticsearch (secured with X-Pack), Kibana
+(secured with X-Pack), and @product@ can communicate effortlessly and securely.
+A Liferay Enterprise Search (LES) subscription is necessary for this
+integration.  Contact 
 [Liferay's Sales department for more information](https://www.liferay.com/contact-us#contact-sales).
 
+| **The Same Monitoring Connector is Used for Elasticsearch 6 and 7**: When first
```
Comment (sez11a) [inline en/deployment/articles-dxp/03-installing-a-search-engine/03-enterprise-search/02-monitoring/01-monitoring-intro.markdown]: Used should not be capitalized.

File: en/deployment/articles/03-installing-a-search-engine/02-elasticsearch/03-installing-elasticsearch.markdown [hunk 3]
```diff
@@ -39,33 +38,30 @@ Installing Elasticsearch takes six steps:
 
 6. Restart @product@ and reindex your search and spell check indexes.
 
-Before continuing, make sure you have set the 
-[`JAVA_HOME` environment variable](https://docs.oracle.com/cd/E19182-01/820-7851/inst_cli_jdk_javahome_t/).
+| **Prerequisites:** Before continuing, make sure you have set the [`JAVA_HOME`
+| environment
+| variable](https://docs.oracle.com/cd/E19182-01/820-7851/inst_cli_jdk_javahome_t/).
```
Comment (sez11a) [inline en/deployment/articles/03-installing-a-search-engine/02-elasticsearch/03-installing-elasticsearch.markdown]: Weird formatting on this link.

File: en/deployment/articles/03-installing-a-search-engine/02-elasticsearch/03-installing-elasticsearch.markdown [hunk 4]
```diff
@@ -39,33 +38,30 @@ Installing Elasticsearch takes six steps:
...
+| 
+| If you have multiple JDKs installed, make sure Elasticsearch and @product@ are
+| using the same version and distribution (e.g., Oracle Open JDK 1.8.0_201). You
+| can specify this in `[Elasticsearch Home]/bin/elasticsearch.in.sh`:
+| `JAVA_HOME=/path/to/java`.
 
-If you have multiple JDKs installed, make sure Elasticsearch and @product@ are
-using the same version and distribution (e.g., Oracle Open JDK 1.8.0_201). You
-can specify this in `[Elasticsearch Home]/bin/elasticsearch.in.sh`:
+Once you perform these steps, you'll have a basic, production-ready instance of
```
Comment (sez11a) [inline en/deployment/articles/03-installing-a-search-engine/02-elasticsearch/03-installing-elasticsearch.markdown]: When you perform these steps, you

File: en/deployment/articles/03-installing-a-search-engine/02-elasticsearch/05-advanced-configuration.markdown [hunk 5]
```diff
@@ -6,7 +6,7 @@ header-id: advanced-configuration-of-the-liferay-elasticsearch-connector
 
 [TOC levels=1-4]
 
-The default configurations for Liferay's Elasticsearch connector module are set
+The default configuration for Liferay's Elasticsearch connector module are set
```
Comment (sez11a) [inline en/deployment/articles/03-installing-a-search-engine/02-elasticsearch/05-advanced-configuration.markdown]: configuration *is* set

File: en/deployment/articles/03-installing-a-search-engine/02-elasticsearch/11-upgrading-to-elasticsearch-73.markdown [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+| **Before Proceeding,** back up your existing data before upgrading
+| Elasticsearch. If something goes wrong during or after the upgrade, roll back
+| to the previous version using the uncorrupted index snapshots. See
+| [here](/docs/7-2/deploy/-/knowledge_base/d/backing-up-elasticsearch) for more
+| information.
+
+| **Known Issue:** See
+| [LPS-103938](https://issues.liferay.com/browse/LPS-103938). The Liferay
+| Connector to Elasticsearch 7 will throw an exception in the log when the LPKG
```
Comment (sez11a) [inline en/deployment/articles/03-installing-a-search-engine/02-elasticsearch/11-upgrading-to-elasticsearch-73.markdown]: throws an exception

File: en/deployment/articles/03-installing-a-search-engine/02-elasticsearch/11-upgrading-to-elasticsearch-73.markdown [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+and click *Execute* for the *Reindex all search indexes* entry.
+
+You must also re-index the spell check indexes.
+
+## Reverting to Elasticsearch 6
+
+Stuff happens. If that stuff involves an unrecoverable failure during the
+upgrade to Elasticsearch 7, roll back to Elasticsearch 6 and regroup.
+
+Since your Elasticsearch 6 and 7 are currently two separate installations, this procedure
```
Comment (sez11a) [inline en/deployment/articles/03-installing-a-search-engine/02-elasticsearch/11-upgrading-to-elasticsearch-73.markdown]: this procedure takes only a few steps: 

(we never, ever say something is "straightforward") https://github.com/liferay/liferay-docs/blob/master/guidelines/03-writers-guidelines.markdown#common-documentation-and-phraseology-issues

**Comment by sez11a** [comment]
@rbohl I went through it here on GitHub because the first thing I saw was a missing link I couldn't provide. Please make all the changes I indicated, and I'll merge this. Thanks!

**Comment by rbohl** [comment]
@sez11a I applied your edits and rebased on master to make sure nothing collided form the LTR docs you already merged. Looks good to me.

**Pull request merged**